### PR TITLE
Serverless: Nightly run on OCP 4.6 for master branch

### PR DESCRIPTION
**Pull request by mgencur**

### CODE REVIEW

File: ci-operator/config/openshift-knative/serverless-operator/openshift-knative-serverless-operator-master__4.6.yaml [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+    cli:
+      paths:
+      - destination_dir: .
+        source_path: /usr/bin/oc
+    openshift_knative-v0.16.0_knative-eventing-src:
+      as:
+      - registry.svc.ci.openshift.org/openshift/knative-v0.16.0:knative-eventing-src
+    openshift_knative-v0.16.0_knative-serving-src:
+      as:
+      - registry.svc.ci.openshift.org/openshift/knative-v0.16.0:knative-serving-src
```
Comment (markusthoemmes) [inline ci-operator/config/openshift-knative/serverless-operator/openshift-knative-serverless-operator-master__4.6.yaml:34]: Do we need these anymore? I think they're obsolete with us adding that to the build image?

Comment (mgencur) [reply ci-operator/config/openshift-knative/serverless-operator/openshift-knative-serverless-operator-master__4.6.yaml:34]: This is some automatic metadata that was added by automation CI tools. We define it manually here: https://github.com/openshift/release/pull/12234/files#diff-3cbc3dcd726d5c860a68e7956c097286R10
Anyway, I think this is required. This is how they get to the serverless-operator-src image, AFAIK.

**Review by markusthoemmes — APPROVED** [review]
/lgtm
/approve

**Pull request merged**